src/components/MapFacetContents.js

```diff
@@ -200,14 +200,29 @@ class MapFacetContents extends React.Component<MapFacetContentsDefaultProps, Map
 
       const points = this.props.buckets.map((bucket) => {
         const value = bucket.value; // JSON.parse(bucket.value);
+        let longitude = NaN;
+        let latitude = NaN;
+        if (typeof value === 'string') {
+          const valueArr = value.split(',');
+          if (valueArr.length === 2) {
+            longitude = Number.parseFloat(valueArr[0]);
+            latitude = Number.parseFloat(valueArr[1]);
+          }
+        } else {
+          longitude = value.longitude;
+          latitude = value.latitude;
+        }
+        if (isNaN(longitude) || isNaN(latitude)) {
```

Comment:
You should call `Number.isNaN()` etc. because ESLint will complain about using the global function `isNaN()`.